mentat/conversation.py

```diff
@@ -92,18 +92,11 @@ async def display_token_count(self):
             )
         else:
             self.max_tokens = context_size
-        if context_size and tokens > context_size:
-            raise MentatError(
-                f"Included files already exceed token limit ({tokens} /"
-                f" {context_size}). Please try running again with a reduced"
-                " number of files."
-            )
-        elif tokens + 1000 > context_size:
+        if context_size and tokens + 1000 > context_size:
             stream.send(
-                f"Warning: Included files are close to token limit ({tokens} /"
-                f" {context_size}), you may not be able to have a long"
-                " conversation.",
-                color="red",
+                f"Included files exceeds token limit ({tokens} / {context_size}). "
+                "Truncating based on task similarity.",
+                color="yellow",
```

Comment:
`display_context()` is called in session before this filtering takes place. I'm not sure we want to print out the full tree if they run `mentat .`. Also we do filtering before a request because we want to know the token count before a request to report to the user. But this is wasteful because ordering by similarity with no prompt isn't meaningful.